cmd/kubeadm/app/cmd/defaults.go

```diff
@@ -76,7 +76,7 @@ func setInitDynamicDefaults(cfg *kubeadmapi.MasterConfiguration) error {
 	// Validate token if any, otherwise generate
 	if cfg.Discovery.Token != nil {
 		if cfg.Discovery.Token.ID != "" && cfg.Discovery.Token.Secret != "" {
-			fmt.Printf("[init] A token has been provided, validating [%s]\n", tokenutil.BearerToken(cfg.Discovery.Token))
+			fmt.Printf("[init] A token has been provided, validating %q\n", tokenutil.BearerToken(cfg.Discovery.Token))
```

Comment:
gotcha, I'm gonna make this log only the ID, we want to minimize the attack area for sure